platform/security/sts/security-sts-pkivalidator/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -23,6 +23,11 @@
             description="The realms to be validated by this validator.">
 	    </AD>
 
+		<AD name="Do Full Path Validation" id="pathValidation" required="true" type="Boolean"
+			default="true"
+			description="Validate the full certificate path. Uncheck to only validate the subject cert.">
```

Comment:
We should mention this path validation is defined in section 6.1 Basic Path Validation in https://tools.ietf.org/html/rfc5280.  Just adding `(RFC5280 6.1)` to the description I think would be enough.